src/frontend/src/defaults/defaultHostList.tsx

```diff
@@ -0,0 +1,19 @@
+import { t } from '@lingui/macro';
+
+import { HostList } from '../states/states';
+
+export const defaultHostList: HostList = {
```

Comment:
At least for now, I would strongly prefer this to be able to test against the current master and stable (sample.* is running the latest stable deployed with packages).
We do not have a config system rn (it is on the epic) and I do not plan to expand the scope of this PR. Keeping multi-host is an explicit goal for me as it enables multi-tenant which is requested regularly.

Reply:
I'm happy with this approach, but we will need an option for locking this down in the server-side configuration, before release.

Reply:
ok. So we could add this as a future task to make it disable-able by a setting. But good idea to enable multi tenant systems even its no real multi tenant because there are still two backend processes required, but its at least one common frontend url.

Reply:
@wolflu05 I have a solution for the backend too - but it is not working with plugins that are not designed for it so not sure how to progress with that. The backends would be hosted on one subdomain per tenant for cookie separation.